jbmc/src/java_bytecode/java_object_factory.cpp

```diff
@@ -1303,7 +1301,7 @@ void java_object_factoryt::gen_nondet_array_init(
     is_valid_java_array(struct_type),
     "Java struct array does not conform to expectations");
 
-  dereference_exprt deref_expr(expr, expr.type().subtype());
+  dereference_exprt deref_expr(expr);
```

Comment:
Braces (as introduced in all the other places)

Reply:
I knew I would forget them somewhere!